### PR TITLE
Feature/player 1350

### PR DESCRIPTION
**Pull request by IvanSakharovskii**

### CODE REVIEW

File: js/controller.js [hunk 1]
```diff
@@ -1268,6 +1270,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
     //called when event listener triggered
     onFullscreenChanged: function() {
+
```
Comment (pilievOoyala) [inline js/controller.js]: Take out this extra line

File: js/components/controlBar.js [hunk 2]
```diff
@@ -114,14 +114,26 @@ var ControlBar = React.createClass({
...
+          this.props.controller.state.isVideoFullScreenSupported) {
+          fullscreen = this.props.controller.state.fullscreen;
+        } else {
+          fullscreen = this.props.controller.state.isFullWindow;
+        }
+
+        if (!fullscreen && typeof this.props.controller.toggleFullscreen === "function") {
+          this.props.controller.toggleFullscreen();
+        }
+        //
```
Comment (pilievOoyala) [inline js/components/controlBar.js]: can you take this out.

**Comment by pilievOoyala** [comment]
LGTM

File: js/components/controlBar.js (hunk 2 above)
Comment (yvinokurov) [reply js/components/controlBar.js]: fix

File: js/controller.js (hunk 1 above)
Comment (yvinokurov) [reply js/controller.js]: fix

**Pull request merged**